mbed_app.json

```diff
@@ -1,8 +1,7 @@
 {
-    "macros": ["MBED_THREAD_STATS_ENABLED"],
     "target_overrides": {
         "*": {
-            "platform.stdio-convert-newlines": 1
+            "platform.thread-stats-enabled": 1
```

Comment:
Can you please use `"true"`?

Reply:
Corrected to match README but the usage of `true` or `1` is inconsistent in the various library config files.